projects/admin/src/app/record/brief-view/acq-account-brief-view/acq-account-brief-view.component.html

```diff
@@ -0,0 +1,29 @@
+ <!--
+  RERO ILS UI
+   Copyright (C) 2019 RERO
+  
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published by
+   the Free Software Foundation, version 3 of the License.
+  
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU Affero General Public License for more details.
+  
+   You should have received a copy of the GNU Affero General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<h5 class="mb-0 d-inline">
+  <a [routerLink]="[detailUrl.link]">
+    {{ record.metadata.name }} - {{ record.metadata.fiscal_year }}
+  </a>
+</h5>
+<div class="card-text">
```

Comment:
Could this `div` be an `article` or a `section`? Semantic tags are better, but if it's not as easy as I would like, we can live with it...

I think the brief view is an article, with a header and an article inside.